tests/neuro_san/zzz_hocons/test_smoke_test_hocons.py

```diff
@@ -27,7 +27,7 @@ class TestSmokeTestHocons(TestCase):
     # We pass it our source file location and a relative path to the common
     # root of the test hocon files listed in the @parameterized.expand()
     # annotation below so the instance can find the hocon test cases listed.
-    DYNAMIC = DynamicHoconUnitTests(__file__, path_to_basis="../../../../fixtures")
+    DYNAMIC = DynamicHoconUnitTests(__file__, path_to_basis="../../fixtures")
```

Comment:
I had moved the file but did not adjust the location where the fixtures are found.